Router

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+
+export default function App() {
+  return (
+    <Router>
+      <div>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/insights">Insights</Link>
+            </li>
+          </ul>
+        </nav>
```

Comment:
Good addition of a navbar! Eventually I would put this into a separate component, but good start!